google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITDownloadToTest.java

```diff
@@ -19,51 +19,102 @@
 import static com.google.common.truth.Truth.assertThat;
 import static org.junit.Assert.fail;
 
+import com.google.cloud.NoCredentials;
 import com.google.cloud.storage.BlobId;
 import com.google.cloud.storage.BlobInfo;
 import com.google.cloud.storage.BucketFixture;
 import com.google.cloud.storage.Storage;
 import com.google.cloud.storage.StorageFixture;
+import com.google.cloud.storage.StorageOptions;
 import com.google.cloud.storage.TestUtils;
+import com.google.cloud.storage.conformance.retry.CleanupStrategy;
+import com.google.cloud.storage.conformance.retry.ParallelParameterized;
+import com.google.cloud.storage.conformance.retry.TestBench;
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.Arrays;
-import org.junit.BeforeClass;
+import org.junit.Before;
 import org.junit.ClassRule;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized.Parameters;
 
+@RunWith(ParallelParameterized.class)
 public final class ITDownloadToTest {
+  @ClassRule(order = 0)
+  public static final TestBench TEST_BENCH =
+      TestBench.newBuilder().setContainerName("it-grpc").build();
 
   @ClassRule(order = 1)
-  public static final StorageFixture storageFixture = StorageFixture.defaultHttp();
+  public static final StorageFixture storageFixtureGrpc =
+      StorageFixture.from(
+          () ->
+              StorageOptions.grpc()
+                  .setHost(TEST_BENCH.getGRPCBaseUri())
+                  .setCredentials(NoCredentials.getInstance())
+                  .setProjectId("test-project-id")
+                  .build());
+
+  @ClassRule(order = 1)
```

Comment:
I'll look this up; but could you add a comment clarifying why you needed to add `@ClassRule(order = x)`?

Reply:
yep can do. The reasoning is to force the rules to be applied in a certain order. storageFixtureGrpc needs the TestBench and the BucketFixtures need their StorageFixtures etc.